{
    "data_folder_path": "data/CVE-2018-21019",
    "pull_requests_urls": [
        "https://github.com/home-assistant/home-assistant/pull/13836"
    ],
    "cve_name": "CVE-2018-21019",
    "repo": "home-assistant/home-assistant",
    "warnings": {
        "cve": "CVE-2018-21019",
        "warnings": []
    },
    "files_extension_stats": {
        ".py": 2
    },
    "file_number": 2,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 49,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}